src/Zenith.sol

```diff
@@ -4,6 +4,9 @@ pragma solidity ^0.8.24;
 import {Passage} from "./Passage.sol";
 
 contract Zenith is Passage {
+    /// @notice Each `transact` call cannot use more than 1/5 of the global `transact` gasLimit for the block.
+    uint256 public constant PER_TRANSACT_GAS_LIMIT = 5;
```

Comment:
still need to align on parameterization of this constant

Reply:
MAX_TRANSACTS_PER_BLOCK

Reply:
well, it wouldn't be max transacts per block right

if this is set at 5, you could still have 10 transacts in the block if they use less than the limit (which i think we expect)

Reply:
when I said "need to align on parameterization" above I meant what should we set this as - 5? 3? 2?